.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
-- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-resource/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
+- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/resource-manager/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
```

Comment:
The repo name here isn't right

Reply:
I think this keeps getting overwritten by the post-processor. I need to look into it more!